README.md

```diff
@@ -75,6 +75,15 @@ In development you can quickly get an environment running using Docker by follow
     1. Ensure that the bot token scope contains at least `users:read`
 1. Run `make up`. This will download dependencies, build and launch a development version of Outline
 
+### Upgrade
+
+
+> The preferred way to run Outline now is through the published docker images which then means the only thing you have to run at upgrade is the migrations
```

Comment:
I'm thinking we need to update this to give actual instructions for docker users, perhaps:

> If you're running Outline with Docker you'll need to run migrations within the docker container, after updating the image. The command will be something like:

```
docker run --rm outlinewiki/outline:latest yarn sequelize:migrate
```

Reply:
I updated the docs to have a docker and yarn sections. Attached is the visual outcome 

<img width="857" alt="Screen Shot 2020-12-25 at 4 52 13 PM" src="https://user-images.githubusercontent.com/13275086/103137429-9b25ff80-46d1-11eb-9a5f-f4a1ca5c5d77.png">